twowaywrapper.ado

```diff
@@ -0,0 +1,102 @@
+clear all
+do twowayreg.ado
+
+
+capture program drop twowayregwrap
+
+program define twowayregwrap, eclass sortpreserve
+version 14 
+syntax varlist(numeric ts fv) [if] [in], [,ABSorb(varlist min=2 max=3) DROP GENerate(name)] [, NEWVars(name) REPLACE] [, ROBUST VCE VCE_2]
+gettoken depvar indepvars : varlist
+_fv_check_depvar `depvar'
+fvexpand `indepvars' 
+marksample touse
+
+if ("`ABSorb('varlist')'"=="`absorb('varlist')'" & "`drop'"=="drop"){
+    twowayset `absorb', drop
+	
+	if ("`NEWVars(`name')'"=="`newvars(`name')'" & "`replace'"==""){
+		projvar `depvar' `indepvars', p(`NEWVars' `varname')
+		local newdepvar w_dist
+		local newindepvar w_mean_np_5 w_x*
+		
+		if("`robust'"=="robust") {
+			twowayreg `newdepvar' `newindepvar', robust
```

Comment:
twowayreg `newdepvar' `newindepvar', `robust' `vce' `vce_2'  

or

let the option be VCE(name)

and then,

twowayreg `newdepvar' `newindepvar', `vce' 

The only problem with this is that we need to define a default if `vce' is not defined.